docs/getting-started.md

```diff
@@ -91,6 +91,7 @@ Currently, we have three different options for a page's layout:
 - **Grid:** ([docs](https://dashboard.flowfuse.com/layouts/types/grid.html)) The default layout for a page. It uses a 12-column grid structure to layout the groups. The widths of each group or widget defines the number of columns they will render in. So, a "width" of 6" would render to 50% of the screen. Grid layouts are entirely responsive, and will adjust to the size of the screen.
 - **Fixed:** ([docs](https://dashboard.flowfuse.com/layouts/types/fixed.html)) Each component will render at a _fixed_ width, no matter what the screen size is. The "width" property is converted a fixed pixel value (multiples of 48px by default).
 - **Notebook:** ([docs](https://dashboard.flowfuse.com/layouts/types/notebook.html)) This layout will stretch to 100% width, up to a maximum width of 1024px, and will centrally align. It's particularly useful for storytelling (e.g. articles/blogs) or analysis type user interfaces (e.g. Jupyter Notebooks), where you want the user to digest content in a particular order through scrolling.
+- **Tabs:** ([docs](https://dashboard.flowfuse.com/layouts/types/tabs.html)) This layout organizes content into separate tabbed sections, allowing users to switch between different views or content categories without leaving the page. Each tab can contain multiple widgets or groups, and it’s specially useful for organizing large amounts of information into easily digestible segments. The "Tabs" layout ensures better navigation and a cleaner UI when multiple content categories need to be displayed in a single view.
```

Comment:
```suggestion
- **Tabs:** ([docs](https://dashboard.flowfuse.com/layouts/types/tabs.html)) This layout organizes content into separate tabbed sections, allowing users to switch between different views or content categories without leaving the page. Each tab can contain multiple groups and widgets, and it’s specially useful for organizing large amounts of information into easily digestible segments. The "Tabs" layout ensures better navigation and a cleaner UI when multiple content categories need to be displayed in a single view.
```

Should this say "groups and widgets" instead of "widgets or groups"?

AFAIK, widgets MUST live in a group (apart from notifications right?)?

Reply:
@Steve-Mcl this updated. thanks for the suggestion.

Reply:
@gayanSandamal not sure if you know but the suggestion i made could have been simply committed by clicking the "Commit Suggestion" button. 

All good though.